tailwind.config.js

```diff
@@ -53,6 +53,9 @@ module.exports = {
         header: "calc(69vh + 4rem)",
         "header-mobile": "calc(80vh + 4rem)",
       },
+      boxShadow: {
+        "profile": "0 2px 12px var(--shadow-2)",
+      },
```

Comment:
cool, but what is --shadow-2? It's referencing a custom variable in css but I don't see it anywhere else

Reply:
this is the facebook shadow you gave me

Reply:
Oui mais Facebook utilisais un style custom comme variable, la c'est juste rien pour nous va trouver ça correspond à quoi